package.json

```diff
@@ -2,33 +2,27 @@
   "name": "html-reporter",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
+  "type": "commonjs",
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build",
+    "build": "tsc && vite build && tsc -p tsconfig.node.cjs.json",
     "preview": "vite preview",
     "lint": "eslint .",
     "lint:fix": "eslint --fix",
-    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
+    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc",
+    "clean": "rm -rf dist index.js index.d.ts *.tsbuildinfo *.tgz"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
why dev `@emotion/react` and other UI lib are part of `devDependencies` and not `dependencies`